app/globals.css

```diff
@@ -7,16 +7,24 @@
   --color-background-light: #e4e4e4;
   --color-primary: #6645bf;
   --color-secondary: #ffb100;
+  --color-chordprinter: #3a3742;
 }
 
-* {
+*,
+:before,
+:after {
   box-sizing: border-box;
   padding: 0;
   margin: 0;
 }
 
+html {
+  font-size: 62.5%;
```

Comment:
tohle nesnasim :D ale kdybychom to ted odstranili, tak by se tim rozmrdaly ty clampy a to by asi byla celkem bolest. Tak ja to pro jednou zkusim prezit teda :D

Reply:
oddelal jsem, zas takova bolest to nebyla :D :D